test/02-device-types.ts

```diff
@@ -0,0 +1,320 @@
+import 'mocha';
+import { app } from '../init';
+import * as _ from 'lodash';
+import { expect } from './test-lib/chai';
+
+import supertest = require('./test-lib/supertest');
+
+describe('device type endpoints', () => {
+	describe('/device-types/v1', () => {
+		it('should succeed to return a result', () => {
+			return supertest(app)
```

Comment:
You can use async/await and vastly improve readability here. See `tests/XX_organizations.ts` in resin-api if you're unsure how.

Reply:
Good point. I migrated it from coffeescript, will refactor it to async/await.